### ISSUE TITLE
Error with Accessing API from trunk version of WP

### ISSUE THREAD

**Issue by regan008** [PressForward/pressforward#866]
```
[01-Nov-2016 16:50:29 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get_field_schema() on null in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php:868
Stack trace:
#0 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(361): WP_REST_Terms_Controller->get_item_schema()
#1 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(315): WP_REST_Controller->get_context_param()
#2 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(882): WP_REST_Controller->get_collection_params()
#3 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(87): WP_REST_Terms_Controller->get_collection_params()
#4 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Controller.php(16): WP_REST_Terms_Controller->register_routes()
#5 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Control in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php on line 868
[01-Nov-2016 16:53:14 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get_field_schema() on null in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php:868
Stack trace:
#0 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(361): WP_REST_Terms_Controller->get_item_schema()
#1 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(315): WP_REST_Controller->get_context_param()
#2 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(882): WP_REST_Controller->get_collection_params()
#3 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(87): WP_REST_Terms_Controller->get_collection_params()
#4 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Controller.php(16): WP_REST_Terms_Controller->register_routes()
#5 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Control in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php on line 868
[01-Nov-2016 16:53:17 UTC] PHP Fatal error:  Uncaught Error: Call to a member function get_field_schema() on null in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php:868
Stack trace:
#0 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(361): WP_REST_Terms_Controller->get_item_schema()
#1 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-controller.php(315): WP_REST_Controller->get_context_param()
#2 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(882): WP_REST_Controller->get_collection_params()
#3 /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php(87): WP_REST_Terms_Controller->get_collection_params()
#4 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Controller.php(16): WP_REST_Terms_Controller->register_routes()
#5 /srv/www/fourpointseven/htdocs/wp-content/plugins/pressforward/Core/API/PF_REST_Terms_Control in /srv/www/fourpointseven/htdocs/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php on line 868
```

**Comment by AramZS** [comment]
Check, should now be resolved on 4.2.x

**Comment by regan008** [comment]
Resolved
